packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -133,9 +133,10 @@ export const RemoteServerFactory = {
 
             // A regex for identifying which SLAS endpoints the custom SLAS private
             // client secret handler will inject an Authorization header.
-            // Do not modify unless a project wants to customize additional SLAS
-            // endpoints that we currently do not support (ie. /oauth2/passwordless/token)
-            applySLASPrivateClientToEndpoints: /\/oauth2\/token/
+            // To allow additional SLAS endpoints, users can override this value in
+            // their project's ssr.js.
+            applySLASPrivateClientToEndpoints:
+                /oauth2\/(token|authorize|passwordless\/(login|token)|password\/(reset|action))/
```

Comment:
Do we need to include `/oauth2/authorize`? 

I don't think that endpoint needs to take in the client secret: https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=authorizeCustomer

```suggestion
/\/oauth2\/(token|passwordless\/(login|token)|password\/(reset|action))/
```